test_utils/factories.py

```diff
@@ -71,6 +78,24 @@ class Meta(object):
     user_id = factory.LazyAttribute(lambda x: FAKER.pyint())
 
 
+class UserDataSharingConsentAuditFactory(factory.django.DjangoModelFactory):
+    """
+    UserDataSharingConsentAuditFactory.
+
+    Creates an instance of UserDataSharingConsentAudit with minimal boilerplate.
+    """
+
+    class Meta(object):
+        """
+        Meta for UserDataSharingConsentAuditFactory.
```

Comment:
https://github.com/open-craft/opencraft/blob/abbe41f/pylintrc#L261-L263 is an example.